src/components/HospitalsTable/index.js

```diff
@@ -28,6 +28,16 @@ const HospitalsTable = ({ hospitals }) => (
             <td className="nhsuk-table__cell">{hospital.wards.length}</td>
             <td className="nhsuk-table__cell">{hospital.bookedVisits}</td>
             <td className="nhsuk-table__cell" style={{ textAlign: "center" }}>
+              <AnchorLink
+                href={`/trust-admin/hospitals/${hospital.id}/edit`}
+                className="nhsuk-u-margin-right-4"
+              >
+                Edit
+                <span className="nhsuk-u-visually-hidden">
+                  {" "}
+                  {hospital.name}
+                </span>
+              </AnchorLink>
```

Comment:
Could we make it so the `View` link appears first? So we're consistent with the wards table so the actions links on table go view, edit and delete.

Reply:
Good spot, done.